server/executor/linter_runner.go

```diff
@@ -100,12 +100,11 @@ func (e *defaultlinterRunner) startWorker() {
 func (e *defaultlinterRunner) onRequest(request LinterRequest) {
 	ctx := request.Context()
 	lintResource := e.analyzerGetter.GetDefault(ctx)
-	linter := linter.NewLinter(lintResource, linter.AvailablePlugins...)
 
-	shouldSkip, reason := linter.ShouldSkip()
+	shouldSkip := lintResource.ShouldSkip()
```

Comment:
The analyzer resource should make the decisions of running or not the linter as it contains the configuration